unit/analyses/disconnect_unreachable_nodes_in_graph.cpp

```diff
@@ -0,0 +1,223 @@
+/*******************************************************************\
+
+Module: Unit test for graph class functions
+
+Author:
+
+\*******************************************************************/
+
+#include <iostream>
+
+#include <testing-utils/catch.hpp>
+
+#include <analyses/call_graph.h>
+#include <analyses/call_graph_helpers.h>
+
+#include <util/std_code.h>
+#include <util/symbol_table.h>
+
+#include <goto-programs/goto_convert_functions.h>
+
+static symbolt
+create_void_function_symbol(const irep_idt &name, const codet &code)
```

Comment:
If these are being duplicated, suggest putting them in `unit/call_graph_test_utils.h` or something like that